spec.md

```diff
@@ -8,121 +8,162 @@ title: Semantic Specification
  [Span D]      +---+-----+
                |         |
-           [Span E]  [Span F]   <-- (2) BAGGAGE ITEM "X" IS AVAILABLE
-                         |              FOR RETRIEVAL BY SPAN F (A
-                +--------+--------+     CHILD OF SPAN C), AS WELL
-                |        |        |     AS SPANS G, H, AND I.
-            [Span G] [Span H] [Span I]
+           [Span E]  [Span F] >>> [Span G] >>> [Span H]
+                                                 ↑
+                                                 ↑
+                                                 ↑
+             (2) BAGGAGE ITEM "X" IS AVAILABLE FOR
+                 RETRIEVAL BY SPAN H (A DESCENDANT OF
+                 SPAN C), AS WELL AS SPANS F AND G.
 ~~~
 
+- Though formally part of the `Tracer` interface, `SpanContext` is essential to [Inject and Extract](#inject-extract) below
 
 ## The `Tracer` Interface
 
 The `Tracer` interface must have the following capabilities:
 
-- Start a new `Span`. There must be a way for the caller to specify a parent `Span`, an explicit start timestamp (other than "now"), and an initial set of `Span` tags. **(py: `start_span`, go: `StartSpanWithOptions`)**
-- `Inject` a `Span` instance into a "carrier" object for cross-process propagation. The type of the carrier is either determined through language reflection or an explicit [format](/propagation#format-identifiers). See the [end-to-end propagation example](/propagation#propagation-example) to make this more concrete.
-- `Join` a Trace given a "carrier" object whose contents crossed a process boundary. `Join` examines the carrier and tries to extract identifying information needed by the new `Span` instance it's trying to create. `Join` also accepts an operation name for that `Span` (since it's typically not included within the carrier). Unless there's an error, `Join` returns a freshly-started `Span` which can be used in the host process like any other. (Note that some OpenTracing implementations consider the `Span`s on either side of an RPC to have the same identity, and others consider the caller to be the parent and the receiver to be the child) The type of the carrier is either determined through language reflection or an explicit [format](/propagation#format-identifiers). See the [end-to-end propagation example](/propagation#propagation-example) to make this more concrete.
-
-Note that `Inject` and `Join` are not entirely symmetrical interfaces since `Span` injection is a fundamentally lossy process (most of the `Span`'s state is meant to be recorded out-of-band from the application's own communication path, whereas Inject-ed state is meant to be sent in-band along with the application data).
-
+- **Start a new `Span`**. The caller can specify zero or more [`SpanContext` references](#references) (e.g., parents or sequence predecessors), an explicit start timestamp (other than "now"), and an initial set of `Span` tags. **(py: `start_span`, go: `StartSpan`)**
```

Comment:
yeah, there are a bunch of misc places that will have to change in the actual PR... just wanted to come to agreement about terminology before I spent time tweaking the .md files.